sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1039,13 +1031,13 @@ SUBROUTINE INITPOST_NETCDF(ncid3d)
       Status=nf90_close(ncid3d)
       deallocate(recname)
 
-! open flux file
-      Status = nf90_open(trim(fileNameFlux),NF90_NOWRITE, ncid2d)
- 
-      if ( Status /= 0 ) then
-        print*,'error opening ',fileNameFlux, ' Status = ', Status
-        print*,'skip reading of flux file' 
-      endif
+!! open flux file
+!      Status = nf90_open(trim(fileNameFlux),NF90_NOWRITE, ncid2d)
+! 
+!      if ( Status /= 0 ) then
+!        print*,'error opening ',fileNameFlux, ' Status = ', Status
+!        print*,'skip reading of flux file' 
+!      endif
```

Comment:
Just curious why to comment out flux file opening?

Reply:
Yeah, I moved the opening of this file to the parent code WRFPOST.f in order to get some attributes earlier on.  I could just delete these lines, but not sure what would happen if the code tried to open the file twice.